src/templates/ObservabilityPackDetails.js

```diff
@@ -37,6 +37,169 @@ const allowedElements = [
   'hr',
 ];
 
+const renderDashboards = (dashboards) => {
+  const content = dashboards.map((dashboard) => (
+    <>
+      <h3>{dashboard.name}</h3>
+      <ImageSlider height={400} images={dashboard.screenshots} />
+      {dashboard.description && (
+        <>
+          <h4>Description</h4>
+          <p>{dashboard.description}</p>
+        </>
+      )}
+    </>
+  ));
+
+  return (
+    <>
+      <Intro
+        css={css`
+          margin-bottom: 16px;
+        `}
+      >
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+        veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea
+        commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+        velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+        occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+        mollit anim id est laborum.
+      </Intro>
+      {content}
+    </>
+  );
+};
+
+const renderAlerts = (alerts) => {
+  const alertContent = alerts.map((alert) => (
```

Comment:
I think react might yell at us here for not providing a `key`.

Reply:
Updated!